_articles/upgrade-pop.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: article
+title: Upgrade to Pop!_OS 18.04 LTS
+description: >
+  Get the newest version of Pop!_OS on your System76 computer! Check out our upgrade directions.
+keywords:
+  - Pop!_OS 17.10
+  - Pop!_OS 18.04
+  - LTS
+  - Upgrade
+  - Update
+  - Release
+  - System76
+  - apt
+image: http://support.system76.com/images/upgrade-ubuntu/Ubuntu-17-04.jpg
+hidden: false
+section: articles
+
+---
+To install Pop!_OS for the first time or get the new recovery system (recommended), download and install from https://system76.com/pop. Though not necessary, we recommend a fresh install with this release for the recovery system. In future releases the recovery partition will download the next version enabling more reliable on-disk release upgrades. The recovery partition also provides an always-stable live environment that helps speed up diagnosing and fixing support issues. 
+
+### Create Installation USB
+
+First, we recommend having Pop!_OS installation media handy before starting any upgrades in case something goes wrong. You can find instructions on creating a a USB with Pop!_OS from [support.system76.com](http://support.system76.com/articles/live-disk/).
+
+### Backup Your Files
+
+Next, we recommend backing up all important files from your computer. The upgrade process will leave your files intact, but it's best to be safe in case something goes wrong. Please read our article on [How to Backup Your Files](articles/backup-files/) for helpful instructions.
+
+### Upgrade
+
+Open a terminal by right-clicking on your desktop background and choose **Open Terminal.**
+
+Type the following command, followed by the enter key.
+
+```
+sudo do-release-upgrade
+```
+
+You'll be prompted to enter your system password, but when you type it, the letters won't show. Just continue typing the password and press <kbd>Enter</kbd>
+
+Type <kbd>Y</kbd> and press enter when prompted
```

Comment:
There will be 3 prompts.  One to install packages, one to remove packages, and one to reboot.  There is also a keyboard layout prompt in some situations.  We should mention that they need to watch the upgrade window and choose 'y' for the 3 prompts.

Reply:
3 'y' prompts and one enter.